src/sentry/sentry_metrics/configuration.py

```diff
@@ -57,4 +73,41 @@ def get_ingest_config(use_case_key: UseCaseKey) -> MetricsIngestConfiguration:
             )
         )
 
-    return _METRICS_INGEST_CONFIG_BY_USE_CASE[use_case_key]
+        _register_ingest_config(
+            MetricsIngestConfiguration(
+                db_backend=IndexerStorage.CLOUDSPANNER,
+                input_topic=settings.KAFKA_INGEST_METRICS,
+                output_topic=settings.KAFKA_SNUBA_GENERICS_METRICS_CS,
+                use_case_id=UseCaseKey.RELEASE_HEALTH,
+                internal_metrics_tag="release-health-spanner",
+                writes_limiter_cluster_options=settings.SENTRY_METRICS_INDEXER_WRITES_LIMITER_OPTIONS,
+                writes_limiter_namespace=RELEASE_HEALTH_CS_NAMESPACE,
+            )
+        )
+
+        _register_ingest_config(
+            MetricsIngestConfiguration(
+                db_backend=IndexerStorage.CLOUDSPANNER,
+                input_topic=settings.KAFKA_INGEST_PERFORMANCE_METRICS,
+                output_topic=settings.KAFKA_SNUBA_GENERICS_METRICS_CS,
+                use_case_id=UseCaseKey.PERFORMANCE,
+                internal_metrics_tag="perf-spanner",
+                writes_limiter_cluster_options=settings.SENTRY_METRICS_INDEXER_WRITES_LIMITER_OPTIONS_PERFORMANCE,
+                writes_limiter_namespace=PERFORMANCE_CS_NAMESPACE,
+            )
+        )
+
+    if db_backend == IndexerStorage.MOCK:
```

Comment:
can you add a code comment here that we're not cachign the mock indexer in order to provide test isolation?